### ISSUE TITLE
Collection of review issues from 10/27/21

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/john-travoltage#401]
- On Audio tab
  - [x] Change "Other Sounds" description to "Play sonifications and sound effects as you interact."
  - [x] Change "Extra Sounds" description to "Play an additional sound or sounds for this sim."
  - [x] Update "Extra Sounds" description to "Play an additional sound or sounds that may be helpful for some learners."
- On Visual tab
  - [x] Change "Interactive Highlights" description to "Add visual highlights for mouse and touch as you interact."
  - [x] The Voice combobox needs voicing instrumentation.
- Toolbar
  - [x] Toolbar chevron button Voicing should ignore Voicing Preferences. 
- Sim Voicing
  - [x] Hearing "Electrons on body" when no Sim Voicing Options are checked, should be muted in this case.
  - [x] Electron discharge context response is never happening
  - [x] Hints for the arm/leg only happen when object responses is checked. Should be independent and happen on its own.

**Comment by terracoda** [comment]
And one more thing about the "Electrons on body" response, it is also heard when  "Context changes" is not checked (i.e., when only "Object information" is checked), but it is technically a context response.

**Comment by terracoda** [comment]
@jessegreenberg, I meant to say, if "Electrons on body" is implemented as a context response, it should only fire when "context changes" checked. I am not 100% sure how it is implemented.

**Comment by jessegreenberg** [comment]
@terracoda can you please verify that these issues have been resolved in https://phet-dev.colorado.edu/html/john-travoltage/1.6.0-preferences.3/phet/john-travoltage_en_phet.html?

**Comment by terracoda** [comment]
I think you got everything @jessegreenberg!
This is safe to close from my perspective.
